homework_1/task_01/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+"flag"
+"fmt"
+"log"
+"strings"
+
+"code-cadets-2021/homework_1/task_01/fizzBuzz"
+)
+
+func parseArgs(start, end int){
+	flag.IntVar(&start, "start", 0, "Start value")
+	flag.IntVar(&end, "end", 0, "End value")
+
+	flag.Parse()
+}
```

Comment:
You're passing `start` and `end` by values to `parseArgs`. This means that `start` and `end` here are completely separate variables from the ones in `main`. You're giving addresses of local `start` and `end` here so the `start` and `end` in `main` are not affected by parsing arguments here whatsoever, they'll always be `0`.

Have you tried running this for some examples? What was the output?

Reply:
It worked, as expected

Reply:
![image](https://user-images.githubusercontent.com/48211138/118468124-bd61fd80-b704-11eb-8243-d5f000b7c702.png)


Tried it both locally on my PC and in Go playground - https://play.golang.org/p/n9j3E-irmPT, doesn't seem to work `start` and `end` are always zero-initialised.

This works though: https://play.golang.org/p/OJvlFWVAXGg